app/controllers/favorites_controller.rb

```diff
@@ -1,6 +1,10 @@
 class FavoritesController < ApplicationController
   include ActionView::Helpers::TextHelper
 
+  def index
+    favorites
```

Comment:
i like that we can just call. favorites now. Good Job!!!

Reply:
:)